src/app/tests/suites/certification/Test_TC_CNET_4_23.yaml

```diff
@@ -0,0 +1,313 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name:
+    12.4.23. [TC-CNET-4.23] [Wi-Fi and Thread] Verification for network
+    interface switching between Wi-Fi and Thread [DUT-Server]
+
+PICS:
+    - CNET.S.F00 && CNET.S.F01
+
+config:
+    nodeId: 0x12344321
+    cluster: "Network Commissioning"
+    endpoint: 0
+
+tests:
+    - label: "Preconditions"
+      verification: |
+          1. DUT supports CNET.S.F00(WI) on endpoint PIXIT.CNET.ENDPOINT_WIFI and CNET.S.F01(TH) on endpoint PIXIT.CNET.ENDPOINT_THREAD
+          2. DUT has a Network Commissioning cluster on endpoint PIXIT.CNET.ENDPOINT_WIFI with FeatureMap attribute of 1, and a Network Commissioning cluster on PIXIT.CNET.ENDPOINT_THREAD with FeatureMap attribute of 2
+          3. DUT is commissioned on PIXIT.CNET.WIFI_1ST_ACCESSPOINT_SSID
+          4. TH has connectivity to both Wi-Fi network (PIXIT.CNET.WIFI_1ST_ACCESSPOINT_SSID) and Thread network (PIXIT.CNET.THREAD_1ST_OPERATIONALDATASET)
+      disabled: true
+
+    - label:
+          "Step 1: TH sends ArmFailSafe command to the DUT with
+          ExpiryLengthSeconds set to 900"
+      verification: |
+          ./chip-tool generalcommissioning arm-fail-safe 900 1 1 0
+
+          Via the TH (chip-tool),  verify that DUT sends ArmFailSafe with timeout as 900 secs to the TH.
+
+          [1725257597.514] [1602947:1602949] [DMG] Received Command Response Data, Endpoint=0 Cluster=0x0000_0030 Command=0x0000_0001
+          [1725257597.514] [1602947:1602949] [TOO] Endpoint: 0 Cluster: 0x0000_0030 Command 0x0000_0001
+          [1725257597.514] [1602947:1602949] [TOO]   ArmFailSafeResponse: {
+          [1725257597.514] [1602947:1602949] [TOO]     errorCode: 0
+          [1725257597.514] [1602947:1602949] [TOO]     debugText:
+          [1725257597.514] [1602947:1602949] [TOO]    }
+      disabled: true
```

Comment:
manual tests are not accepted in the SDK anymore

Reply:
Modified to auto test script